packages/ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -911,18 +911,18 @@ export class AngularCompilerPlugin {
         // When Ivy is enabled we need to add the fields added by NGCC
         // to take precedence over the provided mainFields.
         // NGCC adds fields in package.json suffixed with '_ivy_ngcc'
-        // Example: module -> module__ivy_ngcc
+        // Example: module -> module_ivy_ngcc
         // tslint:disable-next-line:no-any
         (compiler as any).resolverFactory.hooks.resolveOptions
           .for('normal')
           // tslint:disable-next-line:no-any
           .tap('WebpackOptionsApply', (resolveOptions: any) => {
-            const mainFields = (resolveOptions.mainFields as string[])
-              .map(f => [`${f}_ivy_ngcc`, f]);
+            const originalMainFields: string[] = resolveOptions.mainFiels;
```

Comment:
```suggestion
            const originalMainFields: string[] = resolveOptions.mainFields;
```

Reply:
🤦‍♂️

Reply:
The dangers of untyped code :-)

Reply:
The webpack typings are lacking in certain areas unfortunately.